### PR TITLE
Footer Version link to releases page & update TBTC ticker

### LINKED ISSUE #2262: On Earnings page, change tBTC to TBTC since it's a token balance
103314248-c8044a80-4a22-11eb-9bcf-97c2cde1d0b2

This should read "TBTC"

### LINKED ISSUE #2313: Link to Release Notes from dashboard version number
Link to the release notes page from the version number in the bottom lefthand corner of the token dashbaord.

For example,  https://github.com/keep-network/keep-core/releases/tag/token-dashboard%2Fv1.9.0

### PR DESCRIPTION
**Pull request by Battenfield**
- Updates our Keep-core version to a link going out to our releases page, Closes #2313 
- Fixes our currency ticker for TBTC, Closes #2262

### CODE REVIEW

File: solidity/dashboard/src/components/Footer.jsx [hunk 1]
```diff
@@ -18,7 +18,14 @@ const Footer = ({ className }) => {
         <p>&#169; 2020 Keep, SEZC</p>
         <p>All Rights Reserved.</p>
       </div>
-      <div className="footer__app-version">{`Version ${process.env.REACT_APP_VERSION}`}</div>
+      <a 
+        href="https://github.com/keep-network/keep-core/releases"
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/components/Footer.jsx:22]: Also noticed that we have a different text color for `Version...`. Shouldn't be a `@color-grey-60(#6D6D6D)` like the rest? cc @liz-shinn 

obraz

[the remark this answers is not in the document]
Comment (liz-shinn) [reply solidity/dashboard/src/components/Footer.jsx:22]: Yup, it should be! The links above also have a hover action (not sure if that's currently there for the version link in this, but should match)

Screen Shot 2021-03-15 at 12 18 14 PM

[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/components/Footer.jsx:22]: > We use a consistent approach to release versioning, so I think we can redirect directly to the exact release notes. `https://github.com/keep-network/keep-core/releases/tag/token-dashboard/${process.env.REACT_APP_VERSION}`. What do you think?

ahh didn't notice this [commenct](https://github.com/keep-network/keep-core/issues/2313#issuecomment-789096745).

**Review by michalsmiarowski — APPROVED** [review]
🚀

File: solidity/dashboard/src/components/Footer.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply solidity/dashboard/src/components/Footer.jsx]: > Yup, it should be! The links above also have a hover action (not sure if that's currently there for the version link in this, but should match)

Fixed in 8a6155b

**Review by r-czajkowski — APPROVED** [review]

**Pull request merged**